mkt/webapps/indexers.py

```diff
@@ -532,9 +527,18 @@ def get_app_filter(cls, request, additional_data=None, sq=None,
             should = [es_filter.Terms(id=list(set(app_ids)))]
             sq = sq[0:len(set(app_ids))]
 
+        # MUST NOT.
+        must_not = []
+        if data['profile'] and not no_filter:
```

Comment:
Seems like changing this from a double negative would make it clearer. Except `filter` is a keyword.